install/kubernetes/helm/Istio/values.yaml

```diff
@@ -0,0 +1,13 @@
+Istio:
+   auth_enabled: false 
+   initializer_enabled: false
+   mixer_enabled: true
+   pilot_enabled: true
```

Comment:
pilot should always be enabled.  I can't think of a deployed use case where mixer and pilot are not in the system.

Reply:
why?

Reply:
just refreshed and noticed all the answers are already in this review.

Cheers,
-steve